scripts/PHHY-JP/c101111055.lua

```diff
@@ -0,0 +1,108 @@
+--Dogmatikatrix
+local s,id,o=GetID()
+function s.initial_effect(c)
+	--activate
+	local e1=Effect.CreateEffect(c)
+	e1:SetDescription(aux.Stringid(id,0))
+	e1:SetCategory(CATEGORY_SEARCH+CATEGORY_TOHAND)
+	e1:SetType(EFFECT_TYPE_ACTIVATE)
+	e1:SetCode(EVENT_FREE_CHAIN)
+	e1:SetCountLimit(1,id+EFFECT_COUNT_CODE_OATH)
+	e1:SetOperation(s.activate)
+	c:RegisterEffect(e1)
+	--send to gy
+	local e2=Effect.CreateEffect(c)
+	e2:SetDescription(aux.Stringid(id,2))
+	e2:SetCategory(CATEGORY_TOGRAVE)
+	e2:SetType(EFFECT_TYPE_IGNITION)
+	e2:SetRange(LOCATION_SZONE)
+	e2:SetCountLimit(1)
+	e2:SetCondition(s.tgcon)
+	e2:SetTarget(s.tgtg)
+	e2:SetOperation(s.tgop)
+	c:RegisterEffect(e2)
+end
+function s.thfilter(c,specify)
+	return c:IsSetCard(0x145) and (not specify or (c:IsType(TYPE_RITUAL) and c:IsType(TYPE_MONSTER+TYPE_SPELL))) and c:IsAbleToHand()
+end
+function s.activate(e,tp,eg,ep,ev,re,r,rp)
+	if not e:GetHandler():IsRelateToEffect(e) then return end
+	local g=Duel.GetMatchingGroup(s.thfilter,tp,LOCATION_DECK,0,nil,true)
+	if #g>0 and Duel.SelectYesNo(tp,aux.Stringid(id,0)) then
+		Duel.Hint(HINT_SELECTMSG,tp,HINTMSG_ATOHAND)
+		local sg=g:Select(tp,1,1,nil)
+		if #sg>0 and Duel.SendtoHand(sg,nil,REASON_EFFECT)>0 and sg:GetFirst():IsLocation(LOCATION_HAND) then
+			Duel.ConfirmCards(1-tp,sg)
+			local g2=Duel.GetMatchingGroup(s.thfilter,tp,LOCATION_DECK,0,nil)
+			if #g2>0 and Duel.GetFieldGroupCount(tp,0,LOCATION_MZONE)>0 and Duel.SelectYesNo(tp,aux.Stringid(id,1)) then
+				Duel.Hint(HINT_SELECTMSG,tp,HINTMSG_ATOHAND)
+				local sg2=g2:Select(tp,1,1,nil)
+				if #sg2>0 then
+					Duel.BreakEffect()
+					Duel.SendtoHand(sg2,nil,REASON_EFFECT)
+					Duel.ConfirmCards(1-tp,sg2)
+				end
+			end
+		end
+	end
+end
+
+function s.cfilter(c)
+	return c:IsFaceup() and c:IsSetCard(0x145) and c:IsType(TYPE_RITUAL)
+end
+function s.tgcon(e,tp,eg,ep,ev,re,r,rp)
+	return Duel.IsExistingMatchingCard(s.cfilter,tp,LOCATION_MZONE,0,1,nil)
+end
+function s.tgfilter0(c,tp)
+	if c:IsControler(tp) then
+		return c:IsType(TYPE_MONSTER) and c:IsAbleToGrave()
+	else
+		return Duel.IsPlayerCanSendToGrave(tp,c)
```

Comment:
`Duel.IsPlayerCanSendToGrave` does not exist as a function.
```suggestion
		return Duel.IsPlayerCanSendtoGrave(tp,c)
```